src/DssSpell.sol

```diff
@@ -15,216 +16,196 @@
+        DssExecLib.sendPaymentFromSurplusBuffer(DAIF_WALLET,    138_591);
+        DssExecLib.sendPaymentFromSurplusBuffer(SES_WALLET,     155_237);
+
+        // Setup both DssVest modules
+        DssExecLib.authorize(DssExecLib.vat(), MCD_VEST_DAI);
+        DssExecLib.authorize(DssExecLib.getChangelogAddress("GOV_GUARD"), MCD_VEST_MKR);
+        DssVestLike(MCD_VEST_DAI).file("cap", 1 * MILLION * WAD / 30 days);
+        DssVestLike(MCD_VEST_MKR).file("cap", 1_100 * WAD / 365 days);
+        DssExecLib.setChangelogAddress("MCD_VEST_DAI", MCD_VEST_DAI);
+        DssExecLib.setChangelogAddress("MCD_VEST_MKR", MCD_VEST_MKR);
+
+        // DssVestLike(VEST).restrict( Only recipient can request funds
+        //     DssVestLike(VEST).create(
+        //         Recipient of vest,
+        //         Total token amount of vest over period,
+        //         Start timestamp of vest,
+        //         Duration of the vesting period (in seconds),
+        //         Length of cliff period (in seconds),
+        //         Manager address
+        //     )
+        // );
+
+        // Set DAI stream payments
+        DssVestLike(MCD_VEST_DAI).restrict(
+            DssVestLike(MCD_VEST_DAI).create(                                COM_WALLET,   122_700.00 * 10**18, SEP_01_2021, JAN_01_2022 - SEP_01_2021,            0, address(0))
+        );
+        DssVestLike(MCD_VEST_DAI).restrict(
+            DssVestLike(MCD_VEST_DAI).create(                               DAIF_WALLET,   492_971.00 * 10**18, OCT_01_2021, SEP_01_2022 - OCT_01_2021,            0, address(0))
+        );
+        DssVestLike(MCD_VEST_DAI).restrict(
+            DssVestLike(MCD_VEST_DAI).create(                                GOV_WALLET,   123_333.00 * 10**18, SEP_01_2021, OCT_01_2021 - SEP_01_2021,            0, address(0))
+        );
+        DssVestLike(MCD_VEST_DAI).restrict(
+            DssVestLike(MCD_VEST_DAI).create(                                GRO_WALLET,   300_050.00 * 10**18, SEP_01_2021, NOV_01_2021 - SEP_01_2021,            0, address(0))
+        );
+        DssVestLike(MCD_VEST_DAI).restrict(
+            DssVestLike(MCD_VEST_DAI).create(                                MKT_WALLET,   103_134.00 * 10**18, SEP_01_2021, NOV_01_2021 - SEP_01_2021,            0, address(0))
+        );
+        DssVestLike(MCD_VEST_DAI).restrict(
+            DssVestLike(MCD_VEST_DAI).create(                                ORA_WALLET,   196_771.00 * 10**18, SEP_01_2021, JUL_01_2022 - SEP_01_2021,            0, address(0))
+        );
+        DssVestLike(MCD_VEST_DAI).restrict(
+            DssVestLike(MCD_VEST_DAI).create(                                 PE_WALLET, 4_080_000.00 * 10**18, SEP_01_2021, MAY_01_2022 - SEP_01_2021,            0, address(0))
+        );
+        DssVestLike(MCD_VEST_DAI).restrict(
+            DssVestLike(MCD_VEST_DAI).create(                               RISK_WALLET,   184_000.00 * 10**18, SEP_01_2021, SEP_01_2022 - SEP_01_2021,            0, address(0))
```

Comment:
```suggestion
            DssVestLike(MCD_VEST_DAI).create(                               RISK_WALLET, 2_184_000.00 * 10**18, SEP_01_2021, SEP_01_2022 - SEP_01_2021,            0, address(0))
```

Reply:
Same here, thanks. I probably messed it when tabulating the values.

Reply:
Fixed.